src/App.js

```diff
@@ -1,9 +1,23 @@
 import React from "react";
-import Header from "./components/Header";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route} 
+from 'react-router-dom';
+import Confirmation from "./components/Confirmation";
+import Form from "./components/Form";
+import Home from "./components/Home";
 
 const App = () => {
   return (
-  <Header/>
+    <Router>
+      <Switch>
+        <Route exact path='/' component ={Home}/>
```

Comment:
it's possible to use syntax like:
`<Route path='/'><Home></Route>`

This will help in the future when you need to pass props